content/posts/GSoC-week-2.md

```diff
@@ -0,0 +1,43 @@
+---
+
+title: "GSoC week 2 and 3"
+
+date: 2024-06-29
+
+  
+
+---
+
+![GSoC Banner](/GSoC_Banner.png)
+<img src = "/GSoC_Banner.png">
+
+This blog will be about the work I did and interesting stuff I found in these two weeks as a GSoC student.
+
+### Work so far and for the coming weeks.
+
+In these two weeks we heavily planned and implemented lot of Infrastructural changes. These changes may not effect general users of PyBaMM, but 
+developers will surely benefit a lot from them. I also had a taste of how interrelated PR's can be an issue (although not so much in my case), I'll
+explain more about it later.
+
+Let's break down the work :
+
+- Removing `run-tests.py` file : Most python projects uses a `run-tests.py`, which is used to handle command line inputs while running tests. PyBaMM already had 
```

Comment:
```suggestion
- Removing `run-tests.py` file: Most Python projects use a `run-tests.py`, which is used to handle command-line inputs while running tests. PyBaMM already had 
```

Also, not really – not every Python project uses such files. I would say that most projects have used `tox` environments, but it would be good to get some data on this.